functions/src/judge.ts

```diff
@@ -165,6 +165,12 @@ export async function submit(req: Request, res: Response) {
             .json({ error: 'No inputs or expected outputs.' })
         }
 
+        if (inputs.length > MAX_CASES || outputs.length == MAX_CASES) {
```

Comment:
What is the `outputs.length == MAX_CASES` check for? Also I believe the lengths of `input` and `output` should be the same at this point because of the previous if statement.

Reply:
oh thats supposed to be greater than, they should be the same value but I don't trust typescript lol, I can change that